src/weapon/w_043.c

```diff
@@ -21,7 +21,7 @@ s32 func_ptr_80170004(Entity* self) {
         self->facingLeft = PLAYER.facingLeft;
         // this toggles the high bit, but xor does not
         // produce the same set of instructions
-        self->animCurFrame = PLAYER.animCurFrame + 0x8000;
+        self->animCurFrame = PLAYER.animCurFrame + ANIMSET_OVL_FLAG;
```

Comment:
I think this has a different meaning than `ANIMSET_OVL_FLAG`, which is for choosing a gfx bank https://github.com/Xeeynamo/sotn-decomp/blob/6c4e63fd1217b61d39bb978ae32f2c11f9f856a9/src/dra/4A538.c#L374